sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/aggregation/MinAggregator.java

```diff
@@ -15,6 +16,30 @@ public MinAggregator() {
 
     @Override
     public void aggregate(Object item) {
+
+        if (item instanceof ObjectNode) {
+            ObjectNode objectNode = (ObjectNode) item;
+            if (objectNode.hasNonNull("count")) {
```

Comment:
we should make these constants. here and elsewhere

Reply:
done